app/src/main/java/my/noveldokusha/scraper/sources/MoreNovel.kt

```diff
@@ -0,0 +1,133 @@
+package my.noveldokusha.scraper.sources
+
+import my.noveldokusha.data.BookMetadata
+import my.noveldokusha.data.ChapterMetadata
+import my.noveldokusha.scraper.*
+import org.jsoup.nodes.Document
+
+class MoreNovel : SourceInterface.catalog
+{
+    override val name = "More Novel"
+    override val baseUrl = "https://morenovel.net/"
+    override val catalogUrl = "https://morenovel.net/novel/"
+    override val language = "Indonesian"
+    
+    override suspend fun getChapterText(doc: Document): String
+    {
+        val chapter = doc.selectFirst("div.text-left")
```

Comment:
Force not null here, if doesn't find the text is better to fail than replying with empty text.
`        val chapter = doc.selectFirst("div.text-left")!! `